openbb_platform/extensions/devtools/pyproject.toml

```diff
@@ -9,16 +9,19 @@ packages = [{ include = "openbb_devtools" }]
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"               # scipy forces <4.0 explicitly
-ruff = "^0.1.6"
+ruff = "^0.4.4"
 pylint = "^3.0.2"
 mypy = "^1.6.1"
 pydocstyle = "^6.3.0"
-black = "^24.1.0"
+black = "24.1.0"
```

Comment:
This pin won't work because black < 24.3.0 is vulnerable
<img width="932" alt="image" src="https://github.com/OpenBB-finance/OpenBBTerminal/assets/11668535/b0d7652b-b283-430c-a231-d25cfea0c2cc">